test/schedule/health/server_test.exs

```diff
@@ -0,0 +1,40 @@
+defmodule Schedule.Health.ServerTest do
+  use ExUnit.Case
+  import Test.Support.Helpers
+
+  alias Schedule.Health
```

Comment:
I see you're doing this so that you can use `Health.Server` instead of the not-very-specific name `Server`. I probably should have thought of this in my initial comment, but either of these could also be options:
`alias Schedule.Health.Server, as: HealthServer`
`alias Schedule.Health.HealthServer`